sched/definitions.go

```diff
@@ -41,8 +41,12 @@ func DeserializeJob(input []byte) (*Job, error) {
 
 // JobDefinition is the definition the client sent us
 type JobDefinition struct {
-	JobType string
-	Tasks   map[string]TaskDefinition
+	JobType   string
+	Requestor string
+	Tag       string
+	Basis     string
+	Priority  Priority
```

Comment:
I added this because Drew correctly pointed out that I shouldn't be using 'magic' numbers all over the place, so now we have this enum.